jest.config.js

```
@@ -30,7 +30,9 @@ module.exports = {
     "@services/(.*)": "<rootDir>/src/services/$1",
     "^@miniGames(.*)$": ["<rootDir>/src/miniGame/miniGames/$1"],
     "^@miniGameStateService(.*)$": ["<rootDir>/src/miniGame/miniGameStateService/$1"],
+    "@appConfig": "<rootDir>/src/app-config/index.ts",
     "@curiouslearning/analytics": "<rootDir>/node_modules/@curiouslearning/analytics/dist/index.js",
+    "@curiouslearning/core": "<rootDir>/node_modules/@curiouslearning/core/dist/index.js",
```

Biome (2.4.16)

[error] This property is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)